test/lib/test-reporter.js

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2024 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+// This file provides a custom test reporter for the native test runner
+// included in Node.js >=18. The default `spec` reporter writes too much
+// information to be usable in CI, and the `dot` reporter hides which tests
+// failed. This custom reporter outputs nothing for successful tests, and
+// outputs the failing test file when any failing test has occurred.
+//
+// See https://nodejs.org/api/test.html#custom-reporters.
+'use strict'
+
+const { Transform } = require('node:stream')
+const testReporter = new Transform({
+  writableObjectMode: true,
+  transform(event, encoding, callback) {
+    if (event.type !== 'test:fail') {
```

Comment:
I don't think this is working.  I enhanced it to output the file name on successes and see nothing written to stdout.  If we want to use a custom reporter, i'd still like to see progress and obv failures.

Reply:
What change did you make?

Reply:
Also, what issues were you originally referring to? I assumed it was the constant output being rendered to the CI log, but now you're stating you want to see that?